src/styles/fonts.css

```diff
@@ -14,3 +14,29 @@
/sBjwPPBJ5fvFK8Prx5vIt4L/H+4FPhi+Er4VvDd4nvAz8X/wIBFoEwgQOCUoIegm+E1IR8hNqENgidEtYRLhE+IGIlskvkmqiJaJLoCtFTon/EtMSSxCaIvRHnEc8S3weEfyTCJJZIikjGSPZJPpLikJoldU2aT1pP2kP6hvQ3GRuZEpkFMvdkfshGyLbIzpO9IPtETkDOQ26VPJ/8PQUvhTaFQ4psigsUnyjZKbUoGyhnKS9RfqaioeKjUqAyR+WYqpDqPDUmtTS1C+p96j80ZDSmaFzSFNAM05yleUZLTKtM20VHSOeFHo/ePv02Ay2Dc4YhhreMMoxuGAsYexh3Ge8yUTJ5ZBpkesjMxuyLeZh5i/k7iziLc5Zulius2KyyrPZYC9lI2EywnWT7w67E7p59gv06hzCHRY4KjvOcApwmOMs4L3GZ53LA5ZYrl6uRa5brHTcJtwluv9wL3J95lHhc85TynOQl5XXNu82HyafP551viu8M329+AX4n/Hn8m/z3+H8JcAuoC9gR8C3QL4gvaF+wQvCM4F3B30KScMCCkLqQnpBlIQdC7oXyhFqFpoUuCX0QxhVmAIRhYVVA+CJcJNwlvCS8JfxGBE+ETkRARF3EuohbEZ8iuSLTIhdEHoj8FiUR5RNVEDUj6lY0S7ROdEH0rOgt0eeiX8QwAQBRVdMDAAAAAQAAAToAVwAFAAAAAAACACQANAB3AAAAnAF7AAAAAHjahVPLbtNAFD22Q0tLQahCCGU1gq5bF1SQWHSBRItYQFQiuiVO7CSq60ltp21W2fMZ7OAD+AYeO1bwA3wDS87cuQ0BWtWjmXvmvubcO2MAzWAbAdw3v4YiPY4EedzA3RleoLxHa9BY4q6JNcUBvbYVh7Q8URzN4QbHGb7C2B3FC4x4o3gRz5AovooVfFC8RA4fFV+DwXfFK8S/FF8P3gZ3FN9AHL5TfBO3wm+KV7EW/lT8CbejZcWfEUdG8RfqW4q/YjnKPP4RoRmNpq1Ts1vaOq0OzF7aH+edcib30ySzRT3Fe9xHjE08ILuXGKHGEIewWOeYynyNFCUq6i0Kem1SG8uYooVTanZpt4xM6XXA/ZDSoMNZ09JBj5ZDylKsFtl/Z83vXlB2KXNGuDxPue8zQ4EBV8dlQN+aEY+xwWH/ytSl9TJ7m9NzzChz5jzjPJpV0pW1R12CCdeu5JnQ7ti4E4wwPJFKC/H00cf06GnsWCyOs/N2dbk6Kq7r1LTpbcTusveEhxXfkvhY++60LqrP+EI74Dp7UQ2VdH0oFeTC3fNy/gmnr8uff1GOQrrgTrHSgVRqqbSqTFm6uifKxPEzEjWYY9iRF3NCTaJxheQdaw8uu6mNf7p23ns7/w3uEfV5Ti7vbl8Y/Dnfv+E27a6y53JPhuMRZ4wt8tmi9KvRP+ShcM2Ybyx98VXX0gv/b+zM8r/CEb2GtLibzH8DGF+72wAAeNptk2dsG2Ucxn//NrETx87s3ns3jZ04q9OJk+6kzehIW+BiXxyT8116vktJGAWxp0BIfAOxvgBiT4GAD4DYS2wk+MDe4gMgdonvTnEqcdLd732e93nH/V+9TMF5TqUY4H8eaR5/p8hUKWAqBRTiw08RxQQoIUiIUsoop4JKqpjGdGYwk1nMZg5zmcd8FrCQRSxmCUtZxnJWsJJVrGYNa1nHeqrZQA1hItRSR5R6GmikiWY2sonNbGEr24jRQitx2mhnOzvYyS52s4e9dNDJPvbTRTc99HKAgxziMH0c4SjHOIMzOQtFCrmDS7iUp7mJb7iM67iam7mLO8XHVXzMxdwofiniWinmCp7jUwlwC3fzK7/wG7dzLy/zIvfRT4LrSfIqKi/xCm/yGq/zBt+O1+8d3uJt7ifFz9zA+7zLewzyPT9yJWeTZogMGjq3YnCcYUyy2FiMcILvOIcxRjmX8zmPJ7iNk1zAhVzED/zEk1IiQQlJKf/wL6cEKZNyqRCRSqmSaTKd3/mDP/mAz/hcZshMmSWzZY7M5QEelHkyXxbIQv7ib77gS1kki2WJLJVlslxWyEqeklWyWtbIWr7ia1kn66VaNvAhn/CR1EhYIlIrdRKVemmQRmmSZtkom2QzD/Ewj/E4z/MIj/ICl3OPbOEZnpWtso1rJCYt0ipxaZN22S47ZKfskt2yR/ZKh3TKPtkvXdItPdIrB+SgHJLD0idH5Kgc86W00eHBsN/W0zU1NXFX1royVuMx7rIl4tHrb4l6bPLF+k11RPUpDvwxI2Xo6pBfcemLJxTT0H1JF/GEaSiWv80LqV6oze1VXXQoCdtSfbqLDtfUHYQ6B209pZh2RlNsK2RMVr4uN2m66HaHZx0EexJGJqMoiYSqW0FrkvD1uHHLQWGvmdZThXbuG+o9bS37tLX63NnHHAT6koblThcYm2gGuyevmZ0knNJFwjGPLTlGxo/AY9hjxGOtxzqPUY/1Hhs8Nnps8hhzGfbmDYcDA+mUbapJJTvoWpF2l9H2gjbbNIoV0zROaOqAFXBaZjo16Gw2Em1vzbG9xd3cOCNF/YpZrSiaFcrYmpUe1kYdVTys2dnqhJJVS5LpkXRSddyAetxWNMcu1tRs1jGDKVNVLNWdxa9YDktz4zNp3XYzZZqRSicUTTfc7qqUndY0NWNYzubyXnb8xLS8VzmRy/1O3nJiE1Z5v6kkhtR8pjRnqHk5rJiq7sjc1iu8eH6ZMic/STsDXO3UQNVz1Xb6StRMvj04ftVU3S2Dd9Ec8R9GcLZlAAB42mPw3sFwIihiIyNjX+QGxp0cDBwMyQUbGdidtjGEOhoqsjJogTgOPNGMHhwWbBpsEmysHFChGIYgNgcWAxYFFrAQj9M+cQYhBz4HLgc2BwZWBk6gmKDTPgYGBgcoBIsxM7hsVGHsCIzY4NARsZE5xWWjGoi3i6OBgZHFoSM5JAKkJBIIQDZ4cVixabFJsbHyaO1g/N+6gaV3IxODy2bWFDYGFxcAC+8qbAAAAAFXVpj6AAA=)
```

Comment:
why we do this? Can we just add a normal `woff` and `woff2` file? There are plenty of converters out there, right?